lib/link-paths.js

```diff
@@ -1,19 +1,19 @@
 'use babel';
 
-/* eslint-disable no-multi-str, prefer-const*/
+/* eslint-disable no-multi-str, prefer-const, func-names */
 let linkPaths;
-const regex = new RegExp('\
```

Comment:
> if add (?x) flag at the beginning

I don't understand what you mean.

But I can change it back to a multiline regexp if you like. This change  has only stylistic purpose.